src/content/app/tools/blast/components/blast-species-selector/BlastSpeciesSelector.scss

```diff
@@ -0,0 +1,118 @@
+@import 'src/styles/common';
+
+.header {
+  display: flex;
+  width: 860px;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 28px;
+  padding-bottom: 12px;
+}
+
+.headerGroup {
+  display: flex;
+  align-items: center;
+}
+
+.headerTitle {
+  font-size: 15px;
+  font-weight: $bold;
+}
+
+.maxSpecies {
+  font-weight: $light;
+  margin-right: 30px;
+}
+
+.clearAll {
+  margin-right: 30px;
+  color: $blue;
+  cursor: pointer;
+  user-select: none;
+}
+
+.speciesCounter {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 28px;
+  height: 28px;
+  color: $white;
+  background-color: $dark-grey;
+  font-weight: $bold;
+  border-radius: 100%;
+  font-size: 15px;
+  margin: 0 0.6rem 0 0.6rem;
+}
+
+.speciesSelectorContainer {
+  width: 860px;
+  height: 96%;
```

Comment:
This 96% look suspicious.

The effect you are after is probably best achieved through adding `display: flex; flex-direction: column` to the immediate parent of the speciesSelectorContainer and species selector header; and adding `flex-grow: 1` to the speciesSelectorContainer:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/6834224/152983035-f5e1ac05-adb6-453f-935e-0274abcaba45.png">

Reply:
Done